scss/_nav.scss

```diff
@@ -80,6 +81,8 @@
 
 .nav-pills {
   .nav-link {
+    background: none;
```

Comment:
Should the shorthand be used here and the other CSS change?

Also, not sure about `border: 0` / `border: none`

Reply:
we do use the `background` shorthand in a few places (to set things, not to neutralise them, admittedly).

ditto for `border: 0`. we use that in various places. no `border: none`